packages/server/lib/browsers/chrome.coffee

```diff
@@ -315,18 +325,24 @@ module.exports = {
         .then (criClient) =>
           la(criClient, "expected Chrome remote interface reference", criClient)
 
-          ## monkey-patch the .kill method to that the CDP connection is closed
-          originalBrowserKill = launchedBrowser.kill
+          criClient.ensureMinimumProtocolVersion('1.3')
+          .catch (err) =>
+            throw new Error("Cypress requires at least Chrome 64.\n\nDetails:\n#{err.message}")
```

Comment:
curious why you're re-throwing the error here when there's already a specific error being handled by `CDP_VERSION_TOO_OLD`

Reply:
The CDP version warning isn't really actionable to the user, but "upgrade to chrome 64+" is, so I felt it would be best to wrap it with that.